lib/Applify.pm

```diff
@@ -105,6 +105,7 @@ sub import {
   *{$export{documentation}} = sub     { $self->documentation(@_) };
   *{$export{extends}}       = sub     { $self->extends(@_) };
   *{$export{subcommand}}    = sub     { $self->subcommand(@_) };
+  *{$export{SCRIPT}}        = \$self;
```

Comment:
I'm not a fan of this approach, mostly because changing the option_parser is tacked onto a random one of the statements declaring an option.

I think it makes more sense to let the option subroutine calls declare options, and to have a separate call to change the option_parser.

Would it be a bad approach to export `option_parser` so it could be called just like `option`?

`option_parser(Getopt::Long::Parser->new(config => [qw(no_auto_help pass_through bundling)]));`

Reply:
@jhthorsen I get what you are saying about how intuitive the use of the public interface above is. It isn't. It also does not fit with how I use Applify, or how I see you saw Applify being used.

This:
```perl
use Applify getopt_config => [qw{no_auto_help no_pass_through bundling}];

option file => input => 'file to read';
option flag => debug => 'verbose output';

app {
  my $app = shift;
}
```

looks better than:

```perl
use Applify;

option file => input => 'file to read';
option flag => debug => 'verbose output';

$SCRIPT->option_parser->configure(qw{no_auto_help no_pass_through bundling});

app {
  my $app = shift;
}
```

Reply:
I agree, passing getopt_config with use Applify makes sense

Reply:
I really like the above!